swagger_server/itm/itm_scenario_session.py

```diff
@@ -712,15 +712,22 @@ def lookup_probe_response(self, action: Action) -> ProbeResponse:
 
         Args:
             body: The probe response body as a dict.
+        Returns:
+            The probe corresponding to the action, or None if there is no corresponding probe.
         """
+
+        if action.action_id == None or action.action_id == '':
```

Comment:
`if not action.action_id` checks for both

Reply:
+1